dmsg-daemon/cmd/internal/dmsg-client.go

```diff
@@ -0,0 +1,45 @@
+package internal
+
+import (
+	"context"
+	"time"
+
+	"github.com/skycoin/dmsg"
+	"github.com/skycoin/dmsg/cipher"
+	"github.com/skycoin/dmsg/disc"
+	"github.com/skycoin/skywire/pkg/skyenv"
+)
+
+// ConnClient connects to dmsg clients
+func ConnClient() error {
+	var snPK cipher.PubKey
+
+	// convert the pk from string to cipher.PubKey
+	err := snPK.Set(skyenv.DefaultSetupPK)
+	if err != nil {
+		return err
+	}
+
+	cPK, cSK := cipher.GenerateKeyPair()
+
+	// instantiate clients
+	initC := dmsg.NewClient(cPK, cSK, disc.NewHTTP(skyenv.DefaultDmsgDiscAddr), nil)
+	go initC.Serve(context.Background())
+
+	time.Sleep(time.Second)
+
+	// dial responder via DMSG
+	initTp, err := initC.DialStream(context.Background(), dmsg.Addr{PK: snPK, Port: skyenv.DmsgSetupPort})
```

Comment:
I would not allude to Skywire concepts such as `transport` here. We use `dmsg` connections as transports in Skywire, but this is logically separate so conn may be better.